jupyter_notebooks/extraction.ipynb

```diff
@@ -2,21 +2,22 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 1,
    "metadata": {},
    "outputs": [],
    "source": [
-    "import openai"
+    "import openai\n",
+    "import json"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 2,
    "metadata": {},
    "outputs": [],
    "source": [
     "# Set your OpenAI API key\n",
-    "api_key = \"your_api_key_here\""
+    "api_key = \"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\""
```

Comment:
Remove these commits which contains API key.
Put the API key in environment variables